src/activity_type.cpp

```diff
@@ -60,13 +60,16 @@ void activity_type::load( const JsonObject &jo )
     if( jo.has_member( "complex_moves" ) ) {
         result.complex_moves_ = true;
         auto c_moves = jo.get_object( "complex_moves" );
-        result.assistable_     = c_moves.get_bool( "assistable", false );
-        result.bench_affected_ =  c_moves.get_bool( "bench", false );
-        result.light_affected_ =  c_moves.get_bool( "light", false );
-        result.speed_affected_ =  c_moves.get_bool( "speed", false );
-        result.morale_affected_ =  c_moves.get_bool( "morale", false );
-
-        if( c_moves.has_array( "skills" ) ) {
+        result.assistable_ = c_moves.get_bool( "assistable", false );
+        result.bench_affected_ = c_moves.get_bool( "bench", false );
+        result.light_affected_ = c_moves.get_bool( "light", false );
+        result.speed_affected_ = c_moves.get_bool( "speed", false );
+        result.morale_affected_ = c_moves.get_bool( "morale", false );
+
+        c_moves.allow_omitted_members();
+        if( c_moves.has_bool( "skills" ) ) {
+            assign( jo, "skills", result.verbose_tooltip_, false );
```

Comment:
```suggestion
            assign( jo, "skills", result.skill_affected_, false );
```
Is this supposed to be `result.skill_affected_` and not `.verbose_tooltip_`?